src/ray/gcs/gcs_server/grpc_based_resource_broadcaster.cc

```diff
@@ -102,10 +70,7 @@ std::string GrpcBasedResourceBroadcaster::DebugString() {
   return absl::StrCat("GrpcBasedResourceBroadcaster:\n- Tracked nodes: ", node_num);
 }
 
-void GrpcBasedResourceBroadcaster::SendBroadcast() {
-  rpc::ResourceUsageBroadcastData batch;
-  get_resource_usage_batch_for_broadcast_(batch);
-
+void GrpcBasedResourceBroadcaster::SendBroadcast(rpc::ResourceUsageBroadcastData batch) {
```

Comment:
Do we really need a broadcaster btw? It seems to not do much thing now

Reply:
I'm not changing the code for this PR. Just trying to move everything around with minimal change. Later I'll merge everything into syncer (see the prototype).